filters/bevel/src/BevelFilter.js

```diff
@@ -0,0 +1,113 @@
+import {vertex} from '@tools/fragments';
+import fragment from './bevel.frag';
+import * as PIXI from 'pixi.js';
+
+/**
+ * Bevel Filter.<br>
+ * ![original](../tools/screenshots/dist/original.png)![filter](../tools/screenshots/dist/bevel.png)
+ *
+ * @class
+ * @extends PIXI.Filter
+ * @memberof PIXI.filters
+ * @param {number} [rotation = 45] The angle of the light in degrees.
+ * @param {number} [thickness = 2] The tickness of the bevel.
+ * @param {number} [lightColor = 0xffffff] Color of the light.
+ * @param {number} [lightAlpha = 0.7] Alpha of the light.
+ * @param {number} [shadowColor = 0x000000] Color of the shadow.
+ * @param {number} [shadowAlpha = 0.7] Alpha of the shadow.
+ */
+export default class BevelFilter extends PIXI.Filter {
+    constructor(rotation = 45, thickness = 2, lightColor = 0xffffff, lightAlpha = 0.7, shadowColor = 0x000000, shadowAlpha = 0.7){
```

Comment:
Strong preference to do the constructor arguments as an options object. Makes it much easier to use. Other filters with 4+ constructor options tend to do this, especially when the ordering is somewhat arbitrary.